src/mailadm/bot.py

```diff
@@ -36,16 +36,42 @@ def __init__(self, db: DB, account: deltachat.Account):
         self.account = account
         with self.db.read_connection() as conn:
             config = conn.config
-            self.admingrpid = config.admingrpid
-            self.admingroup = account.get_chat_by_id(int(self.admingrpid))
+            self.admingrpid = int(config.admingrpid)
+            self.admingroup = account.get_chat_by_id(self.admingrpid)
 
     @account_hookimpl
     def ac_incoming_message(self, message: deltachat.Message):
         arguments = message.text.split(" ")
-        print("process_incoming command:", arguments)
+        print("process_incoming message:", message.text)
         if not self.check_privileges(message):
-            message.create_chat()
-            message.chat.send_text("Sorry, I only take commands from the admin group.")
+            chat = message.create_chat()
+            if chat.is_group():
+                if message.get_sender_contact() not in self.admingroup.get_contacts():
+                    chat.send_text("Sorry, I'm a strictly non-group bot. You can talk to me 1:1.")
+                    selfcontact = self.account.get_contact_by_addr(self.account.get_config("addr"))
+                    chat.remove_contact(selfcontact)  # leave group
+                elif message.quote:
+                    if message.quote.get_sender_contact().addr == self.account.get_config("addr"):
+                        recipient = chat.get_name()
+                        chat = self.account.create_chat(recipient)
+                        message.set_override_sender_name(self.account.get_config("addr"))
+                        chat.send_msg(message)
```

Comment:
so the incoming message is sent in full to the 1:1 chat?  i am not sure if that is well defined. but i guess you tested it?

Reply:
looks like this:

![Screenshot_2022-10-07_11-50-23](https://user-images.githubusercontent.com/57442838/194526343-02c6e1e5-9aa8-41cf-a5fa-4e61512246b4.png)

I think the way this works at the moment, we could even leave out `set_override_sender_name` as it's overridden anyway. It looked the same or very similar before we left that out. Maybe I'll test that again at some point.